README.md

```diff
@@ -35,16 +35,21 @@ optional arguments:  -h, --help     show this help message and exit
 ## Specific usage
 
 ```bash
-> Python3 checker/AnDOChecker.py tests/ds001/data/Enya
+> python checker/AnDOChecker.py tests/dataset001/Landing
 ```
 
 OR verbose usage
 
 ```bash
-> Python3 checker/AnDOChecker.py -v tests/ds001/data/Enya
+> python checker/AnDOChecker.py -v tests/dataset001/Landing
 
 ```
+OR CLI script
```

Comment:
maybe be a bit more explicit... something like:
"OR, in an equivalent, you can use the command line interface (CLI) provided with the package:"

Reply:
Updated

Reply:
grrrrr, I forgot a word... we need "in an equivalent manner" (sorry)